FormattingStyleGuide.md

```diff
@@ -20,22 +20,32 @@ This document provides formatting and style conventions for publishing documents
 
 ## Table of contents
 - Provide table of contents for longer documents.
-- Use links in the table of contents. Use [GitHub heading anchors](https://gist.github.com/asabaylus/3071099). For example,
-```
-  [Table of contents](StyleManual.md#table-of-contents) 
-```
-links to this section.
+- Use links in the table of contents. Use [GitHub heading anchors](https://gist.github.com/asabaylus/3071099).
+  For example the following code links to this section:
+
+  ```
+  [Table of contents](#table-of-contents) 
+  ```
 
 ## Licence
```

Comment:
Licence vs license, which one is correct? I would suggest American English version which is license.